dotenv_codegen_implementation/src/lib.rs

```diff
@@ -42,6 +42,6 @@ fn expand_env(input_raw: TokenStream) -> TokenStream {
 
     match env::var(var_name) {
         Ok(val) => quote!(#val).into(),
-        Err(_) => panic!("{}", err_msg),
+        _ => panic!("{}", err_msg),
```

Comment:
This seems worse than `Err(_)`, since `Result` will never have more than 2 clear cases. Is this something clippy linted against?

Reply:
yeah, clippy doesn't like `Err(_)`